test/runtests.jl

```diff
@@ -1,4 +1,5 @@
 using OpenCV
 using Base.Test
+using Images
```

Comment:
Currently I'm working on adding the wrappers for template constructors in `src/OpenCV_core.jl` for `Mat`. I'll open a new PR for this. Once this is done and merged only then will I be able to continue with the `test 6` using support of other packages or native methods.